node/node.go

```diff
@@ -205,7 +205,7 @@ func (nn *nexNode) initializeSupervisionTree() error {
 	var err error
 
 	nn.actorSystem, err = goakt.NewActorSystem("nexnode",
-		goakt.WithLogger(logger.NewSlog(nn.options.Logger.Handler().WithGroup("system"))),
+		goakt.WithLogger(logger.NewSlog(nn.options.Logger.Handler().WithGroup("actor_system"))),
```

Comment:
Curious why the addition of the `actor_` prefix?

Reply:
This is to hide the internal goakt logs that we dont want.  There is no other way to turn them off, so I just filter them out and provide a `--show-system-logs` flag to view them if you want

Reply:
Forgot to answer your question....

I added the `actor_` prefix because this PR started using the word "system" and something real and I didn't want the two to be confused

Reply:
[2ab1cef](https://github.com/synadia-io/nex/pull/469/commits/2ab1cefc4bfbf22af78cf44813405821418d4101)